### ISSUE TITLE
insert is blocked

### ISSUE THREAD

**Issue by weickdev** [arangodb/arangodb#3653]
## my environment running ArangoDB
I'm using the latest ArangoDB of the respective release series:
- [ ] 2.8
- [ ] 3.0
- [ ] 3.1
- [x] 3.2
- [ ] self-compiled devel branch

Mode:
- [ ] Cluster
- [ ] Single-Server

Storage-Engine:
- [ ] mmfiles
- [x] rocksdb

On this operating system:
- [ ] DCOS on
  - [ ] AWS
  - [ ] Azure
  - [ ] own infrastructure
- [ ] Linux 
  - [ ] Debian .deb
  - [ ] Ubuntu .deb
  - [ ] SUSE   .rpm
  - [x] RedHat .rpm
  - [ ] Fedora .rpm
  - [ ] Gentoo
  - [ ] docker - official docker library
  - [ ] other:
- [ ] Windows, version: 
- [ ] MacOS, version:

When I insert massive documents, the insert process is blocked. At this time, there many error logs as below:
```
2017-11-10T17:33:11 [27839] ERROR {cluster} cannot create connection to server 'DBServer0002' at endpoint 'tcp://192.168.1.62:18558'
2017-11-10T17:33:11 [27839] ERROR {cluster} cannot create connection to server 'DBServer0002' at endpoint 'tcp://192.168.1.62:18558'
2017-11-10T17:33:11 [27839] ERROR {cluster} cannot create connection to server 'DBServer0002' at endpoint 'tcp://192.168.1.62:18558'
2017-11-10T17:33:11 [27839] ERROR {cluster} cannot create connection to server 'DBServer0002' at endpoint 'tcp://192.168.1.62:18558'
2017-11-10T17:33:11 [27839] ERROR {cluster} cannot create connection to server 'DBServer0002' at endpoint 'tcp://192.168.1.62:18558'
2017-11-10T17:33:11 [27839] ERROR {cluster} cannot create connection to server 'DBServer0002' at endpoint 'tcp://192.168.1.62:18558'
2017-11-10T17:33:11 [27839] ERROR {cluster} cannot create connection to server 'DBServer0002' at endpoint 'tcp://192.168.1.62:18558'
2017-11-10T17:33:11 [27839] ERROR {cluster} cannot create connection to server 'DBServer0002' at endpoint 'tcp://192.168.1.62:18558'
2017-11-10T17:33:11 [27839] ERROR {cluster} cannot create connection to server 'DBServer0002' at endpoint 'tcp://192.168.1.62:18558'
2017-11-10T17:33:11 [27839] ERROR {cluster} cannot create connection to server 'DBServer0002' at endpoint 'tcp://192.168.1.62:18558'
2017-11-10T17:33:11 [27839] ERROR {cluster} cannot create connection to server 'DBServer0002' at endpoint 'tcp://192.168.1.62:18558'
2017-11-10T17:35:10 [27839] ERROR {cluster} ClusterComm::performRequests: got no answer from shard:s5010237:/_db/insert_test/_api/document?collection=s5010237&waitForSync=false&returnNew=false&returnOld=false with error 4
2017-11-10T17:35:10 [27839] ERROR {cluster} ClusterComm::performRequests: got BACKEND_UNAVAILABLE or TIMEOUT from shard:s5010238:/_db/insert_test/_api/document?collection=s5010238&waitForSync=false&returnNew=false&returnOld=false
2017-11-10T17:37:12 [27839] ERROR {cluster} ClusterComm::performRequests: got no answer from shard:s5010237:/_db/insert_test/_api/document?collection=s5010237&waitForSync=false&returnNew=false&returnOld=false with error 4
2017-11-10T17:43:34 [27839] ERROR {cluster} ClusterComm::performRequests: got no answer from shard:s5010231:/_db/insert_test/_api/document?collection=s5010231&waitForSync=false&returnNew=false&returnOld=false with error 4
2017-11-10T17:45:34 [27839] ERROR {cluster} ClusterComm::performRequests: got no answer from shard:s5010231:/_db/insert_test/_api/document?collection=s5010231&waitForSync=false&returnNew=false&returnOld=false with error 4
2017-11-10T17:47:23 [27839] ERROR {cluster} ClusterComm::performRequests: got no answer from shard:s5010232:/_db/insert_test/_api/document?collection=s5010232&waitForSync=false&returnNew=false&returnOld=false with error 4
2017-11-10T17:47:23 [27839] ERROR {cluster} ClusterComm::performRequests: got no answer from shard:s5010232:/_db/insert_test/_api/document?collection=s5010232&waitForSync=false&returnNew=false&returnOld=false with error 4
2017-11-10T17:47:24 [27839] ERROR {cluster} ClusterComm::performRequests: got no answer from shard:s5010232:/_db/insert_test/_api/document?collection=s5010232&waitForSync=false&returnNew=false&returnOld=false with error 4
2017-11-10T17:47:25 [27839] ERROR {cluster} ClusterComm::performRequests: got no answer from shard:s5010232:/_db/insert_test/_api/document?collection=s5010232&waitForSync=false&returnNew=false&returnOld=false with error 4
2017-11-10T17:49:51 [27839] ERROR {cluster} ClusterComm::performRequests: got no answer from shard:s5010232:/_db/insert_test/_api/document?collection=s5010232&waitForSync=false&returnNew=false&returnOld=false with error 4
2017-11-10T17:51:51 [27839] ERROR {cluster} ClusterComm::performRequests: got no answer from shard:s5010238:/_db/insert_test/_api/document?collection=s5010238&waitForSync=false&returnNew=false&returnOld=false with error 4
2017-11-10T17:53:52 [27839] ERROR {cluster} ClusterComm::performRequests: got no answer from shard:s5010238:/_db/insert_test/_api/document?collection=s5010238&waitForSync=false&returnNew=false&returnOld=false with error 4
2017-11-10T17:55:53 [27839] ERROR {cluster} ClusterComm::performRequests: got no answer from shard:s5010238:/_db/insert_test/_api/document?collection=s5010238&waitForSync=false&returnNew=false&returnOld=false with error 4
2017-11-10T17:57:06 [27839] ERROR {cluster} ClusterComm::performRequests: got no answer from shard:s5010238:/_db/insert_test/_api/document?collection=s5010238&waitForSync=false&returnNew=false&returnOld=false with error 4
```
And the primary dbs look like restarting frequently.

**Comment by dothebart** [comment]
Hi, 
this may be related to #3655 ; can you upgrade and revalidate whether the issue still occurs?

**Comment by dothebart** [comment]
Will close this as solved with 3.2.8
